install_cloudnine.sh

```diff
@@ -1,7 +1,9 @@
+# updated to ruby version in gem file
+
 read -p "Enter your cloud9 username: " un
-rvm install ruby-2.1.2
+rvm install ruby-2.3.7
 source $(rvm 2.1.2 do rvm env --path)
```

Comment:
It seems that this line still uses 2.1.2.

Reply:
You are right. I missed that. This is my very first contribution :) 
Thanks for being a friendly place to new contributors. 
I am saving it and will push it.

Reply:
You are very welcome to our community and we look forward to your next contribution :)